### DISCUSSION TITLE
strict tracking decreases star rating on maps with difficult sliders

### DISCUSSION THREAD

**Discussion by Suinollat** [ppy/osu#17481] [Q&A]
While tinkering with the strict tracking mod, I found that it decreases the star rating of the map, while increasing the amount of pp gained. [The map](https://osu.ppy.sh/beatmapsets/537048#osu/1529759) I used has difficult sliders and it seems like it should increase the star rating instead of decreasing it. Star rating dropped slightly from 4.25* to 4.20*, and pp increased from 52 to 63.
version of osu!lazer is 2022.327.0

No Mod
![osu_2022-03-27_07-01-53](https://user-images.githubusercontent.com/85840855/160280896-a01704eb-4a0f-4ac0-8291-22178174c7bb.png)
ST Mod
![osu_2022-03-27_07-01-29](https://user-images.githubusercontent.com/85840855/160280903-459e7d2e-09d6-4dee-80de-db1fc8875f7f.png)

**Comment by apollo-dw** [comment]
Known issue and planned - strict tracking currently has no support from a difficulty calculation standpoint and I am planning to get to it.

It is a little curious that pp increased whilst SR decreased though... I wonder what's going on there
